featurebyte/query_graph/model/table.py

```diff
@@ -639,6 +639,7 @@ class TableDetails(FeatureByteBaseModel):
     """
 
     details: Dict[str, Any] = Field(default_factory=dict)
+    fully_qualified_name: str
```

Comment:
Will this break backward compatibility for existing documents?

Reply:
Looks like this model is not serialized so it should be fine. Curious how will this field be used?

Reply:
this function is currently not yet used in the sdk. Just adding it since it seems to be important table information on how to access a table via SQL, and applicable across the different source types.